lib/src/components/NxTag/NxTag.tsx

```diff
@@ -15,29 +15,28 @@ import NxOverflowTooltip from '../NxTooltip/NxOverflowTooltip';
 import { Props, propTypes, SelectableProps, selectablePropTypes } from './types';
 export { SelectableProps, PublicProps } from './types';
 
-const NxTag: FunctionComponent<Props> =
-    function NxTag(props) {
-      const { children, className, selectedIcons, color, ...attrs } = props,
-          tagClasses = classnames('nx-tag', className, {
-            'nx-selectable-color--indigo': !color, // the default
-            [`nx-selectable-color--${color}`]: color
-          });
+const NxTag = forwardRef<HTMLLabelElement, Props>(function NxTag(props, ref) {
+  const { children, className, selectedIcons, color, title, ...attrs } = props,
+      tagClasses = classnames('nx-tag', className, {
+        'nx-selectable-color--indigo': !color, // the default
+        [`nx-selectable-color--${color}`]: color
+      });
 
-      return (
-        <NxOverflowTooltip>
-          <label className={tagClasses} {...attrs}>
-            <span className="nx-tag__text">{children}</span>
-            {selectedIcons}
-          </label>
-        </NxOverflowTooltip>
-      );
-    };
+  const label = (
+    <label className={tagClasses} title={title} ref={ref} {...attrs}>
+      <span className="nx-tag__text">{children}</span>
+      {selectedIcons}
+    </label>
+  );
+
+  return typeof title === 'undefined' ? <NxOverflowTooltip>{label}</NxOverflowTooltip> : label;
```

Comment:
We have a more robust mechanism for components to detect when they are wrapped in a tooltip: the `TooltipContext` defined in `NxTooltip`.  That should be used instead of checking for `title`.

Reply:
> We have a more robust mechanism for components to detect when they are wrapped in a tooltip: the `TooltipContext` defined in `NxTooltip`. That should be used instead of checking for `title`.

Updated here! 59ced2571cd1548f718660958d93212af59f7860